lib/find.js

```diff
@@ -102,7 +102,13 @@ function _find(tree, _options) {
   var options, root;
 
   if (arguments.length === 1) {
-    var decomposition = decompose(tree);
+    var decomposition;
+
+    if (typeof tree === 'object') {
+      decomposition = decompose(tree.inputTree);
```

Comment:
There is nothing in broccoli land that requires the tree be stashed at `inputTree`.  What if the tree we are provided is nested two levels?  Using this technique we'd have to then do `tree.inputTree.inputTree` (assuming all of the levels use the common name of `inputTree`).